shell/browser/ui/tray_icon_cocoa.mm

```diff
@@ -19,8 +20,8 @@
 #include "ui/native_theme/native_theme.h"
 
 @interface StatusItemView : NSView {
-  electron::TrayIconCocoa* trayIcon_;       // weak
-  ElectronMenuController* menuController_;  // weak
+  raw_ptr<electron::TrayIconCocoa> trayIcon_;  // weak
+  ElectronMenuController* menuController_;     // weak
```

Comment:
why is this not a `raw_ptr` ?

Reply:
@deepak1556 https://source.chromium.org/chromium/chromium/src/+/main:base/memory/raw_ptr.md under the section titled `When it is okay to continue using raw C++ pointers -> Unsupported cases leading to compile errors`, it lists `Pointers to Objective-C objects`, which is what ElectronMenuController is.